src/utils/viewer/viewer.cpp

```diff
@@ -394,6 +405,15 @@ Viewer::Viewer(const Arguments& arguments)
   };
   agentConfig.sensorSpecifications[0]->resolution =
       esp::vec2i(viewportSize[1], viewportSize[0]);
+  if (args.isSet("orthographic")) {
+    // try orthographic camera
+    agentConfig.sensorSpecifications[0]->sensorSubType =
+        esp::sensor::SensorSubType::Orthographic;
+
+  } else {
+    agentConfig.sensorSpecifications[0]->sensorSubType =
+        esp::sensor::SensorSubType::Pinhole;
+  }
```

Comment:
```suggestion
  agentConfig.sensorSpecifications[0]->sensorSubType =
      args.isSet("orthographic") ?
      esp::sensor::SensorSubType::Orthographic : 
      esp::sensor::SensorSubType::Pinhole;
```

Sorry, code golf, won't do this again :)